package.json

```diff
@@ -40,6 +41,7 @@
     "string-length": "2.0.0"
   },
   "devDependencies": {
+    "@types/micro": "7.3.3",
```

Comment:
canary of micro ships typings: https://github.com/zeit/micro/commit/167f32deb37b389bd47864a2d9c80353f65577b0